gpkit/nomials/nomial_math.py

```diff
@@ -739,9 +739,51 @@ def as_posyslt1(self):
 
     def as_gpconstr(self, x0):
         "Returns GP approximation of an SP constraint at x0"
+        print "x0", x0
         siglt0, = self.unsubbed
         x0 = self._fill_default_x0(x0, siglt0.varlocs)
         posy, negy = siglt0.posy_negy()
         mec = (posy.mono_lower_bound(x0) == negy.mono_lower_bound(x0))
         mec.substitutions = self.substitutions
         return mec
+
+    def process_result(self, result):
+        "Checks that all constraints are satisfied with equality"
```

Comment:
the docstring isn't quite correct, because what this currently does is set `result["sigeq_logsumerror"]`, and the only check related to that is in `signomial_program` and slated for removal before merging